packages/xo-web/src/common/intl/messages.js

```diff
@@ -77,6 +77,7 @@ const messages = {
   withMemory: 'With memory',
   offline: 'Offline',
   noLicenseAvailable: 'No license available',
+  emailPlaceholderExample: 'Email address, e.g: it@enterprise.com',
```

Comment:
https://github.com/vatesfr/xen-orchestra/search?q=company.net&unscoped_q=company.net
```suggestion
  emailPlaceholderExample: 'Email address, e.g.: it@company.net',
```